library/src/com/google/maps/android/geojson/GeoJsonLayer.java

```diff
@@ -71,6 +74,69 @@ public GeoJsonLayer(GoogleMap map, int resourceId, Context context)
     }
 
     /**
+     * Sets a single click listener for the entire GoogleMap object, that will be called
+     * with the corresponding GeoJsonFeature object when an object on the map (Polygon,
+     * Marker, Polyline) is clicked.
+     *
+     * Note that if multiple GeoJsonLayer objects are bound to a GoogleMap object, calling
+     * setOnFeatureClickListener on one will override the listener defined on the other. In
+     * that case, you must define each of the GoogleMap click listeners manually
+     * (OnPolygonClickListener, OnMarkerClickListener, OnPolylineClickListener), and then
+     * use the GeoJsonLayer.getFeature(mapObject) method on each GeoJsonLayer instance to
+     * determine if the given mapObject belongs to the layer.
+     *
+     * @param listener GeoJsonOnFeatureClickListener object providing the onFeatureClick
```

Comment:
You don't need to specify the type for the param in javadoc - it's apparent from the name. Please remove all through this change.